src/sprout/Controllers/Admin/OperatorCategoryAdminController.php

```diff
@@ -154,8 +154,9 @@ public function _editSave($item_id)
         $update_fields['name'] = $_POST['name'];
         $update_fields['access_operators'] = (int) (bool) @$_POST['access_operators'];
         $update_fields['access_noapproval'] = (int) (bool) @$_POST['access_noapproval'];
-        $update_fields['access_reportemail'] = (int) (bool) @$_POST['access_reportemail'];
+        $update_fields['access_reportemail'] = (int) (bool) @$_POST['access_reportemail']; // System reports
         $update_fields['access_homepage'] = (int) (bool) @$_POST['access_homepage'];
+        $update_fields['access_email_report'] = (int) (bool) @$_POST['access_email_report']; // Custom reports
```

Comment:
Is there no way to make these names clearer?

Like, I appreciate the comments but it wouldn't it be better if we just fix the names so they're self-evident? Otherwise we would need to carry around these alt-name comments wherever we go.

Reply:
OK only just noticed that the first already exists and we don't want a migration on our hands.

Or do we..? Because we've actually got three, if we include the multiedit fields:

- access_reportemail
- access_email_report
- access_report

Reply:
@gwillz This is a good question about naming. I was rather aware at the time that it's a bit of a mess with the different report/email types we have. This one didn't sit in any of the existing permissions, so I just tried to keep it the same naming as the rest of the new feature. 

Not sure it's worth biting off a big migration or a breaking change but I'd certainly be open to making it clearer, happy to chat more about this one!

Reply:
@jamiemonksuk OK I reckon we can leave any migration plans for now. Perhaps we can merge these two things in future if it's too confusing.

Looking deeper into `access_reportemail` - it's only used for the link checker and was obviously left open to permit other reports to piggy back the flag. I wouldn't say it's a "system report" but it's definitely a hard-coded one unlike this new custom reports feature.

I think what's confusing here for me is the alternate descriptions:

- `// Custom reports`
- `'access_email_report' => 'Can create custom email reports',`
- `'m_access_email_report' => 'Automate email reports',`

When actually it's all the same feature, so yeah I'll mush all those together and I think we can merge this.